content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -245,6 +245,57 @@ There is also a blue banner at the top of the quotation template design with a l
 return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation form in the
 back-end of the *Sales* application.
 
+Mass cancel quotations/sales orders
+===================================
+
+Cancel multiple quotations (or sales orders) at once from the list view of the
+:guilabel:`Quotations` page.
+
+First navigate to the :guilabel:`Quotations` page by going to :menuselection:`Sales app
+--> Orders --> Quotations`, or open the **Sales** app from the main Odoo dashboard.
+
+The :guilabel:`Quotations` page displays all the quotations and sales order in a default list view.
+
+.. image:: quote_template/quotations-page.png
+   :align: center
+   :alt: The main Quotations page in the Odoo Sales application, in a default list view.
+
+From here, select the desired quotations (or sales orders) from the list, by ticking the checkbox
+on the far-left of their respective line. To select *all* the items on the list, tick the checkbox
+to the far-left of the column titles, and all the items become selected.
+
+The total number of selected items are displayed at the top of the page.
```

Comment:
Can be more succinct: 17 lines --> 7
Simple instructions do not need so much breadth. 

```suggestion
Cancel multiple quotations (or sales orders) by navigating to the :menuselection:`Sales app -->
Orders --> Quotations` dashboard, landing, by default, in the list view. Then on the left side of
the table, tick the preferred checkboxes for removal.

..tip::
  Select all records in the table by selecting the checkbox column header at the top-left of the
  table; the total number of selected items are displayed at the top of the page.
```